ext/node/polyfills/http.ts

```diff
@@ -552,6 +549,13 @@ class ClientRequest extends OutgoingMessage {
 
         if (span) {
           span.setAttribute("http.response.status_code", res.status);
+          if (res.status >= 400) {
+            span.setAttribute("error.type", String(res.status));
+            span.setStatus({
+              code: 2, // Error
+              message: res.statusText,
```

Comment:
You can omit the message:

> Don’t set the span status description if the reason can be inferred from http.response.status_code.

https://opentelemetry.io/docs/specs/semconv/http/http-spans/